pymc3/distributions/continuous.py

```diff
@@ -194,7 +210,7 @@ def logp(self, value):
                      tau > 0, sd > 0)
 
 
-class HalfNormal(PositiveContinuous):
+class HalfNormal(PositiveUnivariateContinuous):
```

Comment:
I think we should go that route here unless it would cause other unanticipated problems.